File: mycollab-services-ondemand/src/main/java/com/mycollab/ondemand/module/support/service/EmailReferenceService.java
package com.mycollab.ondemand.module.support.service;

import com.mycollab.db.persistence.service.IService;

/**
 * @author MyCollab Ltd
 * @since 5.3.5
 */
public interface EmailReferenceService extends IService {
    void save(String email);

    void remove(String email);
}

File: mycollab-services-ondemand/src/main/java/com/mycollab/ondemand/module/support/service/EmailReferenceService.kt
package com.mycollab.ondemand.module.support.service

import com.mycollab.db.persistence.service.IService

/**
 * @author MyCollab Ltd
 * @since 5.3.5
 */
interface EmailReferenceService : IService {
    fun save(email: String)

    fun remove(email: String)
}
